tasty-query/shared/src/main/scala/tastyquery/Symbols.scala

```diff
@@ -583,6 +583,17 @@ object Symbols {
     /** Is this symbol an exporter generated by an `export` statement? */
     final def isExport: Boolean = flags.is(Exported)
 
+    /** Is this symbol a method with at least one parameter with a default value? */
+    final def hasParamsWithDefault: Boolean =
```

Comment:
Should we use a singular here, as in "has at least one param with default" ?

```suggestion
    final def hasParamWithDefault: Boolean =
```

Reply:
Indeed, that's probably better. Changed.